lrauv_ignition_plugins/worlds/buoyant_tethys.sdf

```diff
@@ -41,16 +42,8 @@
           <density>1</density>
         </density_change>
       </graded_buoyancy-->
+      <uniform_fluid_density>1000</uniform_fluid_density>
     </plugin>
-
-    <!-- Requires ParticleEmitter2 in ign-gazebo 4.8.0, which will be copied
-      to ParticleEmitter in Ignition G.
-      See https://github.com/ignitionrobotics/ign-gazebo/pull/730 -->
-    <plugin
-      filename="ignition-gazebo-particle-emitter2-system"
-      name="ignition::gazebo::systems::ParticleEmitter2">
```

Comment:
Are we removing the turbidity_generator block below? If we keep that block, then this block will still be needed. They haven't copied the `ParticleEmitter2` over to `ParticleEmitter` yet, as far as I can see. Even after that, one of them is still needed - this block will just need to be renamed.

Reply:
We aren't using it right now. I'll keep this in but keep it commented.